docs/data/charts/label/label.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Charts - Label
+productId: x-charts
+components: BarChart, ScatterChart, LineChart, PieChart
+---
+
+# Charts - Label
+
+<p class="description">Label refers to the text reference of a series or data. It is used in many components like the "legend", "tooltip",  and inside an "arc" of a pie chart.</p>
+
+## Basic display
+
+In the most basic example, you can pass in a `string` as a series' label, and it will be rendered like that by all the different components that use it in order to differentiate each series.
+
+{{"demo": "BasicLabel.js"}}
+
+## Conditional formatting
+
+In order to change the content of the label based on where it is being displayed. You can pass a function to the `label` property of the [BarSeriesType](/x/api/charts/bar-series-type/), [LineSeriesType](/x/api/charts/line-series-type/) and [ScatterSeriesType](/x/api/charts/scatter-series-type/) or [PieSeriesType](/x/api/charts/pie-series-type/)`.data.label` in case of a pie chart.
+
+### Bars, Lines and Scatter
+
+The [Bars](/x/react-charts/bars/), [Lines](/x/react-charts/lines/) and [Scatter](/x/react-charts/scatter/) charts all work in the same way regarding the `label` property.
+
+Each of the `series` can receive a different `function` or `string` as their `label`.
+
```

Comment:
I'm wondering if we could consider that as the default behavior (This will probably be similar with radar chart for example), and we add sections for edge cases (for now we only have pie as an edge case)

The structure would be 

- Basic display
- Conditional formatting
- Pie

and later if we have more edge cases

- Basic display
- Conditional formatting
- Particular cases
  - Pie

```suggestion
```

If not, at lease this section does not need to introduce back the fact that `label accept `string` and `function`. It's already said in the previous section. Better to focus on how this function can be used

Reply:
yeah I think it makes a lot of sense, applied your suggestions there and also proofread the doc to tie up all the suggested changes.